e2e/old-arch/various-suite/navigation.spec.ts

```diff
@@ -1,43 +0,0 @@
-import { e2e } from '../utils';
-import { fromBaseUrl } from '../utils/support/url';
-
-describe('Docked Navigation', () => {
```

Comment:
deleted this from the old-arch (that tests old dashboard arch), as there is nothing in in the old dashboard architecture that relates / changes how mega menu and mega menu docking works